assets/src/apps/authoring/components/HeaderNav.tsx

```diff
@@ -116,6 +118,33 @@ const HeaderNav: React.FC<HeaderNavProps> = (props: HeaderNavProps) => {
                 </button>
               </span>
             </OverlayTrigger>
+            {isAdmin && (
+              <OverlayTrigger
+                placement="bottom"
+                delay={{ show: 150, hide: 150 }}
+                overlay={
+                  <Tooltip id="button-tooltip" style={{ fontSize: '12px' }}>
+                    View Page Revision History (Admin)
```

Comment:
🎨 Seems like `Revision History` could work and be less wordy.

Reply:
ok